sympy/parsing/latex/lark/transformer.py

```diff
@@ -0,0 +1,426 @@
+import re
+
+import sympy
+from sympy.parsing.latex.errors import LaTeXParsingError
+
+import lark
```

Comment:
You may need to guard the import with
lark = import_module(‘lark’)

Reply:
I've made the change.